lib/node/rpc.js

```diff
@@ -717,15 +718,20 @@ class RPC extends RPCBase {
 
   async getBlockFilter(args, help) {
     if (help || args.length < 1 || args.length > 2)
-      throw new RPCError(errs.MISC_ERROR, 'getblockfilter "hash"');
+      throw new RPCError(errs.MISC_ERROR, 'getblockfilter "hash" ( "type" )');
```

Comment:
huh, funny that the rpc already allowed up to 2 arguments, probably left over from when filter type was added, and then removed. now its back!